setup.py

```diff
@@ -55,6 +55,15 @@ def is_requirement(line):
     slug = fextra.split("/")[1]
     EXTRAS_REQUIRE[slug] = load_requirements(fextra)
 
+# To run tests & linting across the entire repo, we need to install the union
+# of *all* extra requirements lists *plus* the dev-specific requirements.
+# If this list contains conflicting pins, then installing it will fail;
+# that is intentional. 
+EXTRAS_REQUIRE["dev"] = sorted({
+    *load_requirements("dev-requirements.txt"),
```

Comment:
It looks like we want to [move away from using this file](https://github.com/openedx/repo-tools/blob/master/dev-requirements.txt), so I think we should just reference `development.txt` directly.
```suggestion
    *load_requirements("requirements/development.txt"),
```

Reply:
Good point 👍🏻